go/vt/vtgate/vindexes/vschema_test.go

```diff
@@ -899,7 +899,7 @@ func TestVSchemaMirrorRules(t *testing.T) {
 		name: "stfu1",
```

Comment:
surprised there are no unit tests here. shouldn't we add at least that adds a view and checks the global routing?

Reply:
I did add to vschema_manager_test which does it

Reply:
I will add here as well